src/mixins/filing-template-mixin.ts

```diff
@@ -463,19 +473,17 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
         ...draftFiling.continuationIn.contactPoint,
         confirmEmail: draftFiling.continuationIn.contactPoint.email
       })
-    } else {
-      this.setBusinessContact({ ...EmptyContactPoint })
```

Comment:
If contact point is not in the draft, we should be able to leave the initial value in the store (which is EmptyContactPoint).